app/android/build.gradle

```diff
@@ -3,6 +3,7 @@ buildscript {
     repositories {
         google()
         mavenCentral()
+        maven { url 'https://jitpack.io' }
```

Comment:
Probably not a huge deal, but do we actually need this one as well? Since the deps in question aren't part of the actual buildscript running.